src/wp-includes/html-api/class-wp-html.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class WP_HTML {
+	public static function make_tag( $tag_name, $attributes = null, $data = '' ) {
+		$is_void = WP_HTML_Spec::is_void_element( $tag_name );
+		$html = $is_void ? "<{$tag_name}>" : "<{$tag_name}>{$data}</{$tag_name}>";
+
+		$p = new WP_HTML_Tag_Processor( $html );
```

Comment:
I considered it and it might make sense to construct it. this is more of an exploration of sharing the tag processor for this utility. I think initially I was looking at returning the processor itself, which would make it possible to modify after creating. the thought of that was for conditional attributes, but here that's already possible by setting an attribute to `false`